docs/clients/uavcan_node.rst

```diff
@@ -0,0 +1,132 @@
+path. This allows the creation of a UavcanNodeClient object. See the Message Table below for available messages.
+All message strings use the Short Names. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the UavcanNodeClient is:
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+    % Make a UavcanNodeClient object with obj_id 0
+    uavcanNode = UavcanNodeClient(’com’,com);
+    % Use the UavcanNodeClient object
+    uavcanNodeId = uavcanNode.get(’uavcan_node_id’);
+
+Python
+~~~~~~
+
+To use the UAVCAN Node Client in Python, import ``iqmotion`` and create a module that has the Arming Handler Client within its firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the UAVCAN Node Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0)
+    
+    uavcan_node_id = |variable_name|.get("uavcan_node", "uavcan_node_id") 
+    print(f"UAVCAN Node ID: {uavcan_node_id}")
+
+Message Table
+~~~~~~~~~~~~~
+
+Type ID 80 | UAVCAN Node
+
++--------+------------------------+----------------+-----------+------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Sub ID | Short Name             | Access         | Data Type | Unit             | Note                                                                                                                                                                                                                                                                                                                    |
++========+========================+================+===========+==================+=========================================================================================================================================================================================================================================================================================================================+
+| 0      | uavcan_node_id         | get, set, save | uint32    | Number [1, 127]  | This ID is used by the module to uniquely identify itself on the DroneCAN bus. The module must be power cycled after changing the ID. Each module on the bus must have a unique ID. While the IDs can be a number between 1 and 127, it is recommended to avoid using 1 as this is commonly used by flight controllers. |
```

Comment:
The module must be power cycled in order for a new ID to take effect. I think that might be clearer